### ISSUE TITLE
test failure on i386 (and other architectures)

### ISSUE THREAD

**Issue by olebole** [astropy/photutils#322]
The `aperture_photometry` DocTest fails on a number of platforms when building for Debian:

```
____________________ [doctest] docs/photutils/aperture.rst _____________________
116 whether its center is in or out of the aperture.  For ``'subpixel'``,
117 pixels are divided into a number of subpixels, which are in or out of
118 the aperture based on their centers.
119 
120 This example uses the ``'subpixel'`` method where pixels are resampled
121 by a factor of 5 in each dimension::
122 
123     >>> phot_table = aperture_photometry(data, apertures,
124     ...                                  method='subpixel', subpixels=5)
125     >>> print(phot_table['aperture_sum'])
Differences (unified diff with -expected +actual):
    @@ -2,4 +2,4 @@
     <BLANKLINE>
     ------------
    -       27.96
    -       27.96
    +       28.12
    +       28.12

/tmp/photutils-test-ZC86KN/docs/photutils/aperture.rst:125: DocTestFailure
```

This happens on [i386](https://buildd.debian.org/status/fetch.php?pkg=photutils&arch=i386&ver=0.2.1-1&stamp=1454584160), [arm64](https://buildd.debian.org/status/fetch.php?pkg=photutils&arch=arm64&ver=0.2.1-1&stamp=1454584212), [powerpc](https://buildd.debian.org/status/fetch.php?pkg=photutils&arch=powerpc&ver=0.2.1-1&stamp=1454584378), [pp64el](https://buildd.debian.org/status/fetch.php?pkg=photutils&arch=ppc64el&ver=0.2.1-1&stamp=1454584191), [s390x](https://buildd.debian.org/status/fetch.php?pkg=photutils&arch=s390x&ver=0.2.1-1&stamp=1454584306). On amd64, it [builds fine](https://buildd.debian.org/status/fetch.php?pkg=photutils&arch=amd64&ver=0.2.1-1&stamp=1454584343) however.

**Comment by bsipocz** [comment]
Thank @olebole for the report and the logs. I'll try to figure out what's going on.

**Comment by larrybradley** [comment]
@olebole This appears to be fixed in #475.  The tests now pass on 32-bit machines (see https://circleci.com/gh/larrybradley/photutils/4).  Please re-open this issue if you still find failing tests.

Our 32-bit `CircleCI` testing will become active soon (#476).
